fingerprint2.js

```diff
@@ -498,6 +500,56 @@
         done(keys);
       }, 1);
     },
+	audioKey: function(keys, done){
+		if ((!this.options.excludeAudio) && !(("ontouchstart" in window) || (navigator.maxTouchPoints > 0) || (navigator.msMaxTouchPoints > 0))) {
```

Comment:
How would you rate the entropy of the plugin adding this audio fg? Would it be unique??